Improve percentiles agg documentation (#9797)
opensearch-project/documentation-website: _aggregations/metric/percentile-ranks.md

  }
 }
}
```

> This response means the value `10` is at the `5.5`th percentile, and the value `15` is at the `8.3`rd percentile. 

Reviewer: ```suggestion
This response indicates that the value `10` is at the `5.5`th percentile and the value `15` is at the `8.3`rd percentile. 
```

> Like in the percentiles aggregation, you can control the level of approximation using the optional `tdigest.compression` field. A larger value means the data structure that approximates percentiles is more accurate, but uses more heap space. The default value is 100. 

Reviewer 2: ```suggestion
As with the `percentiles` aggregation, you can control the level of approximation by setting the optional `tdigest.compression` field. A larger value increases the precision of the approximation but uses more heap space. The default value is 100.
```

Reviewer 2: @kolchfa-aws Done.

> For example, to set compression to 200: 

Reviewer: This would ideally be a complete sentence 😄
